source/sonata_simulation.rst

```diff
@@ -81,7 +81,7 @@ Parameters defining global simulation settings. As NEURON is the engine used for
    dt                              float      Mandatory   Duration of a single integration timestep. Given in ms.
    random_seed                     integer    Mandatory   For random sequences, seed is a positive integer added in order to give the user the capacity to change the sequences.
    spike_threshold                 float      Optional    The spike detection threshold. A spike is detected whenever the voltage in the spike detection location goes over the spike threshold value. Default is -30.0 mV. `NEURON specific details <https://nrn.readthedocs.io/en/latest/python/modelspec/programmatic/network/netcon.html#NetCon.threshold>`_.
-   integration_method              text       Optional    Selects the NEURON/CoreNEURON `integration method <https://nrn.readthedocs.io/en/latest/python/simctrl/programmatic.html#secondorder>`_. This parameter sets the NEURON global variable h.secondorder. The allowed values are ‘0’ (default) for fully implicit backward euler, ‘1’ for Crank-Nicolson and ‘2’ for Crank-Nicolson with fixed ion currents.
+   integration_method              integer    Optional    Selects the NEURON/CoreNEURON `integration method <https://nrn.readthedocs.io/en/latest/python/simctrl/programmatic.html#secondorder>`_. This parameter sets the NEURON global variable h.secondorder. The allowed values are 0 (default) for fully implicit backward euler, 1 for Crank-Nicolson and 2 for Crank-Nicolson with fixed ion currents.
```

Comment:
Minor typo, it's "Nicholson"

Reply:
I correct myself, "Nicolson" is correct, it's the NEURON docs that have it wrong :p

Reply:
I prefer names, it's less cryptic and the relation to NEURON's `secondorder` is clear from the documentation.